src/main/java/xogame/models/Board.java

```diff
@@ -37,13 +37,24 @@ public Board() {
         }
 
         return columns;
-    }    
+    }
 
     public List<List<Integer>> getDiagonals() {
         List<List<Integer>> diagonals = new ArrayList<>();
-        
-        diagonals.add(Arrays.asList(board[0], board[4], board[8]));
-        diagonals.add(Arrays.asList(board[2], board[4], board[6]));
+
```

Comment:
Why do you think this is better than the previous version /where we just hardcode the indices/?